chainlink-modsec/libmodsec/pkg/modsectypes/types.go

```diff
@@ -179,3 +179,17 @@ type Hasher interface {
 	// needs to be included alongside message data.
 	Hash(ctx context.Context, data []byte) ([]byte, error)
 }
+
+type VerifierType int
+
+const (
+	VerifierTypeCLCommit VerifierType = iota // CL Commit verifier
+	VerifierTypeUSDC                         // USDC verifier
+	VerifierTypeLBTC                         // LBTC verifier
+)
```

Comment:
I feel like this shouldn't be a thing, reasoning is that in a permissionless setup you won't be able to keep up with new verifier "types" being created. Also you could have multiple instances of the same verifier type, e.g. a finalized commit, ftf commit, etc.